src/tasks/WasmAppBuilder/WasmAppBuilder.csproj

```diff
@@ -13,13 +22,32 @@
   </ItemGroup>
 
   <Target Name="PublishBuilder"
-          AfterTargets="Build"
-          DependsOnTargets="Publish" />
+          AfterTargets="Build">
+
+    <!-- needed for publishing with multi-targeting. We are publishing essentially to get the SR.MetadataLoadContext.dll :/ -->
+    <ItemGroup>
+      <_PublishFramework Include="$(TargetFrameworks)" />
+    </ItemGroup>
+    <MSBuild Projects="$(MSBuildProjectFile)" Targets="Publish" Properties="TargetFramework=%(_PublishFramework.Identity)" />
```

Comment:
This looks a bit scary. Aren't the dependencies that need to be packaged included in the build output folder?

Reply:
In case of net472, yes, but not for net6.0 :|